modules/ts/CMakeLists.txt

```diff
@@ -47,3 +47,18 @@ if(OPENCV_DISABLE_THREAD_SUPPORT)
   # described in `ts_gtest.h`.
   ocv_target_compile_definitions(${the_module} PUBLIC GTEST_HAS_PTHREAD=0)
 endif()
+
+if(BUILD_FUZZ_TESTS)
+  include(FetchContent)
```

Comment:
`FetchContent` is available since cmake 3.11, OpenCV@4.x currently has 3.5.1 minimal cmake version requirement. Can we use external library as dependency? I can see that it is part of oss-fuzz-base/base-builder image.